src/mme-app/monitor_subscriber.c

```diff
@@ -139,7 +159,12 @@ handle_monitor_processing(void *message)
 	            handle_imsi_list_req(mir, sock_fd);
                 break;
             }
-            
+        case 2:
```

Comment:
how do you send this case 2 ?

Reply:
We send the type in exporter python file. Not added to that file yet. Can be added when needed.